driver/src/models/standing_order.rs

```diff
@@ -0,0 +1,30 @@
+use serde_derive::{Deserialize};
+
+#[derive(Clone, Deserialize, Eq, Ord, PartialEq, PartialOrd)]
+#[serde(rename_all = "camelCase")]
+pub struct StandingOrder {
+    pub account_id: u16,
+    orders: Vec<super::Order>,
+}
+
+impl From<mongodb::ordered::OrderedDocument> for StandingOrder {
+    fn from(document: mongodb::ordered::OrderedDocument) -> Self {
+        let account_id = document.get_i32("_id").unwrap() as u16;
+        StandingOrder {
+            account_id,
+            orders: document
+                .get_array("orders")
+                .unwrap()
+                .iter()
+                .map(|o| o.as_document().unwrap())
```

Comment:
picky situation are the `o` on this line and the next meant to be interpreted as the same thing? If not I might call them something different. Either way, I think we could expand/elaborate on what these are here (since we don't necessarily need to save characters here).